cms/templates/js/mock/mock-paged-container-xblock.underscore

```diff
@@ -0,0 +1,261 @@
+<header class="xblock-header">
+    <div class="xblock-header-primary">
+        <div class="header-details">
+            <span class="xblock-display-name">Test Container</span>
+        </div>
+        <div class="header-actions">
+            <ul class="actions-list">
+            </ul>
+        </div>
+    </div>
+</header>
+<article class="xblock-render">
+    <div class="xblock" data-locator="locator-container" data-request-token="page-render-token"
+         data-init="MockXBlock" data-runtime-class="StudioRuntime" data-runtime-version="1">
+
+         <script type="text/template" id="paging-header-tpl">
+		    <div class="meta-wrap">
+		    <div class="meta">
+		        <%= messageHtml %>
+		    </div>
+		    <nav class="pagination pagination-compact top">
+		        <ol>
+		            <li class="nav-item previous"><a class="nav-link previous-page-link" href="#"><i class="icon fa fa-angle-left"></i> <span class="nav-label">Previous</span></a></li>
+		            <li class="nav-item next"><a class="nav-link next-page-link" href="#"><span class="nav-label">Next</span> <i class="icon fa fa-angle-right"></i></a></li>
+		        </ol>
```

Comment:
all icon fonts should have an ```aria-hidden="true"``` attribute on them. Because links like this are often consumed by screen reader users out of context, its nice to add screen reader text that adds the context:
```
<a>Previous <span class="sr">Page</span></a>
<a>Next <span class="sr">Page</span></a>
```